packages/replay/src/index.ts

```diff
@@ -124,6 +124,11 @@ export class Replay implements Integration {
    */
   private stopRecording: ReturnType<typeof record> | null = null;
 
+  /**
+   * We overwrite `client.recordDroppedEvent`, but store the original method here so we can restore it.
+   */
+  private _originalRecordDroppedEvent?: Client['recordDroppedEvent'];
```

Comment:
I have been omitting the underscore prefix with private members since it's marked as private already but since this seems to be a pattern in js sdk, maybe we should update the replay sdk to follow this pattern.

Reply:
Yes, this is def. part of the todo list! :)